materialize-google-pubsub/transactor.go

```diff
@@ -11,7 +12,13 @@ import (
 )
 
 type transactor struct {
-	topics []*pubsub.Topic
+	bindings []*topicBinding
+}
+
+type topicBinding struct {
+	identifier string
```

Comment:
Are the `identifier` and `topicName` fields used separately anywhere? If not it might be more efficient to move the whole `fmt.Sprintf` bit out into setup and just use the same string each time.

Reply:
I thought about this some more and concluded that the `topicName` is totally redundant to include in the attribute to the topic, since all messages published to a topic will have the same topic name. So I was able to simplify this an remove the string formatting entirely.